src/bejeweled/board/SpecialType.java

```diff
@@ -0,0 +1,30 @@
+package bejeweled.board;
+
+import java.util.EnumMap;
+import java.util.Random;
+
+import javafx.scene.image.Image;
+
+/**
+ * @author Timo This class defines the different types of gems.
+ */
+public enum SpecialType {
+	NORMAL, DOUBLE, CROSS;
+
+	private static SpecialType[] allspecialgems = {NORMAL, DOUBLE, CROSS};
+
+	/**
+	 * @param s,
+	 *            the String that is used.
+	 * @return The GemType for a certain String.
+	 */
+	public static SpecialType typeFromString(String s) {
```

Comment:
theres a build in method valueOf in Enum, which you can use instead of calling this method.
SpecialType.ValueOf("Normal") Wil return SpecialType.NORMAL